### PR TITLE
update weights

### PR DESCRIPTION
**Pull request by coderobe**
🏋️‍♀️

### CODE REVIEW

File: runtime/kusama/src/weights/pallet_scheduler.rs [hunk 1]
```diff
@@ -50,9 +50,9 @@ impl<T: frame_system::Config> pallet_scheduler::WeightInfo for WeightInfo<T> {
 	// Storage: Scheduler Lookup (r:0 w:1)
 	/// The range of component `s` is `[1, 50]`.
 	fn on_initialize_periodic_named_resolved(s: u32, ) -> Weight {
-		Weight::from_ref_time(1_256_000 as u64)
-			// Standard Error: 42_000
-			.saturating_add(Weight::from_ref_time(26_925_000 as u64).saturating_mul(s as u64))
+		Weight::from_ref_time(11_217_000 as u64)
```
Comment (EgorPopelyaev) [inline runtime/kusama/src/weights/pallet_scheduler.rs:53]: x10 jump

File: runtime/kusama/src/weights/pallet_scheduler.rs [hunk 2]
```diff
@@ -64,9 +64,9 @@ impl<T: frame_system::Config> pallet_scheduler::WeightInfo for WeightInfo<T> {
 	// Storage: Scheduler Lookup (r:0 w:1)
 	/// The range of component `s` is `[1, 50]`.
 	fn on_initialize_named_resolved(s: u32, ) -> Weight {
-		Weight::from_ref_time(921_000 as u64)
-			// Standard Error: 35_000
-			.saturating_add(Weight::from_ref_time(21_922_000 as u64).saturating_mul(s as u64))
+		Weight::from_ref_time(15_840_000 as u64)
```
Comment (EgorPopelyaev) [inline runtime/kusama/src/weights/pallet_scheduler.rs:67]: x15 jump

File: runtime/kusama/src/weights/pallet_scheduler.rs [hunk 3]
```diff
@@ -77,9 +77,9 @@ impl<T: frame_system::Config> pallet_scheduler::WeightInfo for WeightInfo<T> {
 	// Storage: Preimage StatusFor (r:1 w:1)
 	/// The range of component `s` is `[1, 50]`.
 	fn on_initialize_periodic_resolved(s: u32, ) -> Weight {
-		Weight::from_ref_time(0 as u64)
-			// Standard Error: 62_000
-			.saturating_add(Weight::from_ref_time(24_926_000 as u64).saturating_mul(s as u64))
+		Weight::from_ref_time(10_761_000 as u64)
```
Comment (EgorPopelyaev) [inline runtime/kusama/src/weights/pallet_scheduler.rs:80]: jump from 0 -> 10_761_000

File: runtime/kusama/src/weights/pallet_staking.rs [hunk 4]
```diff
@@ -400,9 +400,9 @@ impl<T: frame_system::Config> pallet_staking::WeightInfo for WeightInfo<T> {
 	// Storage: System BlockWeight (r:1 w:1)
 	/// The range of component `v` is `[500, 1000]`.
 	fn get_npos_targets(v: u32, ) -> Weight {
-		Weight::from_ref_time(0 as u64)
-			// Standard Error: 40_000
-			.saturating_add(Weight::from_ref_time(7_752_000 as u64).saturating_mul(v as u64))
+		Weight::from_ref_time(310_326_000 as u64)
```
Comment (EgorPopelyaev) [inline runtime/kusama/src/weights/pallet_staking.rs:403]: jump from 0 ->  310_326_000

File: runtime/kusama/src/weights/runtime_common_crowdloan.rs [hunk 5]
```diff
@@ -123,9 +123,9 @@ impl<T: frame_system::Config> runtime_common::crowdloan::WeightInfo for WeightIn
 	// Storage: System Account (r:2 w:2)
 	/// The range of component `n` is `[2, 100]`.
 	fn on_initialize(n: u32, ) -> Weight {
-		Weight::from_ref_time(0 as u64)
-			// Standard Error: 43_000
-			.saturating_add(Weight::from_ref_time(51_696_000 as u64).saturating_mul(n as u64))
+		Weight::from_ref_time(15_720_000 as u64)
```
Comment (EgorPopelyaev) [inline runtime/kusama/src/weights/runtime_common_crowdloan.rs:126]: jump from 0 -> 15_720_000

File: runtime/kusama/constants/src/weights/block_weights.rs [hunk 6]
```diff
@@ -42,23 +42,24 @@ use frame_support::{
...
 	///
 	/// Percentiles nanoseconds:
-	///   99th: 6_332_047
-	///   95th: 6_308_225
-	///   75th: 6_236_204
-	pub const BlockExecutionWeight: Weight = WEIGHT_PER_NANOS.saturating_mul(6_192_341);
+	///   99th: 390_162
+	///   95th: 383_320
+	///   75th: 356_698
+	pub const BlockExecutionWeight: Weight = WEIGHT_PER_NANOS.saturating_mul(356_760);
```
Comment (ggwpez) [inline runtime/kusama/constants/src/weights/block_weights.rs:57]: uhhh what happend here

File: runtime/polkadot/src/weights/pallet_bounties.rs [hunk 7]
```diff
@@ -128,9 +128,9 @@ impl<T: frame_system::Config> pallet_bounties::WeightInfo for WeightInfo<T> {
 	// Storage: System Account (r:2 w:2)
 	/// The range of component `b` is `[1, 100]`.
 	fn spend_funds(b: u32, ) -> Weight {
-		Weight::from_ref_time(0 as u64)
-			// Standard Error: 28_000
-			.saturating_add(Weight::from_ref_time(29_706_000 as u64).saturating_mul(b as u64))
+		Weight::from_ref_time(13_392_000 as u64)
```
Comment (EgorPopelyaev) [inline runtime/polkadot/src/weights/pallet_bounties.rs:131]: jump from 0 -> 13_392_000

File: runtime/polkadot/src/weights/pallet_collective_council.rs [hunk 8]
```diff
@@ -16,160 +16,202 @@
...
+	/// The range of component `m` is `[4, 100]`.
+	/// The range of component `p` is `[1, 100]`.
+	/// The range of component `m` is `[4, 100]`.
+	/// The range of component `p` is `[1, 100]`.
 	fn close_early_disapproved(m: u32, p: u32, ) -> Weight {
-		Weight::from_ref_time(39_515_000 as u64)
+		Weight::from_ref_time(30_247_000 as u64)
 			// Standard Error: 0
-			.saturating_add(Weight::from_ref_time(165_000 as u64).saturating_mul(m as u64))
+			.saturating_add(Weight::from_ref_time(25_000 as u64).saturating_mul(m as u64))
```
Comment (EgorPopelyaev) [inline runtime/polkadot/src/weights/pallet_collective_council.rs:139]: x7 jump down

File: runtime/polkadot/src/weights/pallet_collective_council.rs [hunk 9]
```diff
@@ -16,160 +16,202 @@
...
+	/// The range of component `p` is `[1, 100]`.
 	fn close_early_approved(b: u32, m: u32, p: u32, ) -> Weight {
-		Weight::from_ref_time(54_757_000 as u64)
+		Weight::from_ref_time(39_632_000 as u64)
 			// Standard Error: 0
-			.saturating_add(Weight::from_ref_time(3_000 as u64).saturating_mul(b as u64))
+			.saturating_add(Weight::from_ref_time(1_000 as u64).saturating_mul(b as u64))
 			// Standard Error: 0
-			.saturating_add(Weight::from_ref_time(163_000 as u64).saturating_mul(m as u64))
+			.saturating_add(Weight::from_ref_time(27_000 as u64).saturating_mul(m as u64))
```
Comment (EgorPopelyaev) [inline runtime/polkadot/src/weights/pallet_collective_council.rs:160]: x7 jump down

File: runtime/kusama/src/weights/pallet_scheduler.rs (hunk 1 above)
Comment (ggwpez) [reply runtime/kusama/src/weights/pallet_scheduler.rs:53]: image
Looks like an asymptotic decrease actually.

File: runtime/polkadot/src/weights/pallet_scheduler.rs [hunk 10]
```diff
@@ -77,9 +77,9 @@ impl<T: frame_system::Config> pallet_scheduler::WeightInfo for WeightInfo<T> {
 	// Storage: Preimage StatusFor (r:1 w:1)
 	/// The range of component `s` is `[1, 50]`.
 	fn on_initialize_periodic_resolved(s: u32, ) -> Weight {
-		Weight::from_ref_time(125_000 as u64)
-			// Standard Error: 37_000
-			.saturating_add(Weight::from_ref_time(23_169_000 as u64).saturating_mul(s as u64))
+		Weight::from_ref_time(13_757_000 as u64)
```
Comment (EgorPopelyaev) [inline runtime/polkadot/src/weights/pallet_scheduler.rs:80]: significant jump 125_000 -> 13_757_000

File: runtime/polkadot/src/weights/pallet_staking.rs [hunk 11]
```diff
@@ -400,9 +400,9 @@ impl<T: frame_system::Config> pallet_staking::WeightInfo for WeightInfo<T> {
 	// Storage: System BlockWeight (r:1 w:1)
 	/// The range of component `v` is `[500, 1000]`.
 	fn get_npos_targets(v: u32, ) -> Weight {
-		Weight::from_ref_time(0 as u64)
-			// Standard Error: 37_000
-			.saturating_add(Weight::from_ref_time(7_545_000 as u64).saturating_mul(v as u64))
+		Weight::from_ref_time(105_025_000 as u64)
```
Comment (EgorPopelyaev) [inline runtime/polkadot/src/weights/pallet_staking.rs:403]: significant jump 0 -> 105_025_000

File: runtime/polkadot/src/weights/runtime_common_crowdloan.rs [hunk 12]
```diff
@@ -123,9 +123,9 @@ impl<T: frame_system::Config> runtime_common::crowdloan::WeightInfo for WeightIn
 	// Storage: System Account (r:2 w:2)
 	/// The range of component `n` is `[2, 100]`.
 	fn on_initialize(n: u32, ) -> Weight {
-		Weight::from_ref_time(0 as u64)
-			// Standard Error: 41_000
-			.saturating_add(Weight::from_ref_time(49_451_000 as u64).saturating_mul(n as u64))
+		Weight::from_ref_time(21_631_000 as u64)
```
Comment (EgorPopelyaev) [inline runtime/polkadot/src/weights/runtime_common_crowdloan.rs:126]: jump from 0 -> 21_631_000

File: runtime/rococo/src/weights/runtime_common_crowdloan.rs [hunk 13]
```diff
@@ -123,9 +123,9 @@ impl<T: frame_system::Config> runtime_common::crowdloan::WeightInfo for WeightIn
 	// Storage: System Account (r:2 w:2)
 	/// The range of component `n` is `[2, 100]`.
 	fn on_initialize(n: u32, ) -> Weight {
-		Weight::from_ref_time(0 as u64)
-			// Standard Error: 25_000
-			.saturating_add(Weight::from_ref_time(51_756_000 as u64).saturating_mul(n as u64))
+		Weight::from_ref_time(18_891_000 as u64)
```
Comment (EgorPopelyaev) [inline runtime/rococo/src/weights/runtime_common_crowdloan.rs:126]: jump 0 -> 18_891_000

File: runtime/westend/src/weights/pallet_election_provider_multi_phase.rs [hunk 14]
```diff
@@ -109,11 +109,11 @@ impl<T: frame_system::Config> pallet_election_provider_multi_phase::WeightInfo f
 	/// The range of component `a` is `[500, 800]`.
 	/// The range of component `d` is `[200, 400]`.
 	fn elect_queued(a: u32, d: u32, ) -> Weight {
-		Weight::from_ref_time(9_172_000 as u64)
+		Weight::from_ref_time(0 as u64)
```
Comment (EgorPopelyaev) [inline runtime/westend/src/weights/pallet_election_provider_multi_phase.rs:112]: significant change

File: runtime/westend/src/weights/runtime_common_crowdloan.rs [hunk 15]
```diff
@@ -61,24 +61,24 @@ impl<T: frame_system::Config> runtime_common::crowdloan::WeightInfo for WeightIn
...
 			.saturating_add(T::DbWeight::get().reads(4 as u64))
 			.saturating_add(T::DbWeight::get().writes(4 as u64))
 	}
 	// Storage: Skipped Metadata (r:0 w:0)
 	/// The range of component `k` is `[0, 500]`.
 	fn refund(k: u32, ) -> Weight {
-		Weight::from_ref_time(0 as u64)
-			// Standard Error: 14_000
-			.saturating_add(Weight::from_ref_time(18_621_000 as u64).saturating_mul(k as u64))
+		Weight::from_ref_time(19_773_000 as u64)
```
Comment (EgorPopelyaev) [inline runtime/westend/src/weights/runtime_common_crowdloan.rs:79]: jump from 0 -> 19_773_000

File: runtime/westend/src/weights/runtime_common_crowdloan.rs [hunk 16]
```diff
@@ -123,9 +123,9 @@ impl<T: frame_system::Config> runtime_common::crowdloan::WeightInfo for WeightIn
 	// Storage: System Account (r:2 w:2)
 	/// The range of component `n` is `[2, 100]`.
 	fn on_initialize(n: u32, ) -> Weight {
-		Weight::from_ref_time(0 as u64)
-			// Standard Error: 28_000
-			.saturating_add(Weight::from_ref_time(48_794_000 as u64).saturating_mul(n as u64))
+		Weight::from_ref_time(18_538_000 as u64)
```
Comment (EgorPopelyaev) [inline runtime/westend/src/weights/runtime_common_crowdloan.rs:126]: jump from 0 -> 18_538_000

File: runtime/kusama/constants/src/weights/block_weights.rs (hunk 6 above)
Comment (coderobe) [reply runtime/kusama/constants/src/weights/block_weights.rs:57]: this change is consistent across runtimes, did anything change in the benchmark logic?

**Comment by chevdor** [comment]
I am surprised not to see Kusama in [here](https://weights.tasty.limo/compare?old=v0.9.28&new=92191b301b6607d989bfce7dd45552907501472c&threshold=90&unit=weight&repo=polkadot&path_pattern=runtime%2F**%2Fsrc%2Fweights%2F**%2F*.rs&method=guess-worst&ignore_errors=true&git_pull=false)

File: runtime/kusama/constants/src/weights/block_weights.rs (hunk 6 above)
Comment (ggwpez) [reply runtime/kusama/constants/src/weights/block_weights.rs:57]: I have no idea whats going on.

Comment (coderobe) [reply runtime/kusama/constants/src/weights/block_weights.rs:57]: aye, will do

**Comment by ggwpez** [comment]
> I am surprised not to see Kusama in [here](https://weights.tasty.limo/compare?old=v0.9.28&new=92191b301b6607d989bfce7dd45552907501472c&threshold=90&unit=weight&repo=polkadot&path_pattern=runtime%2F**%2Fsrc%2Fweights%2F**%2F*.rs&method=guess-worst&ignore_errors=true&git_pull=false)

The benchmarking has some shortcomings for very short functions, like `Remark`. Remark has an empty function body and can therefore mess with the results. https://github.com/paritytech/substrate/pull/11806 should improve precision.

File: runtime/kusama/constants/src/weights/block_weights.rs (hunk 6 above)
Comment (coderobe) [reply runtime/kusama/constants/src/weights/block_weights.rs]: reverted that change

**Review by ggwpez — APPROVED** [review]

**Pull request merged**